pkg/shell/superuser.jsx

```diff
@@ -108,7 +110,16 @@
 
     function init() {
         return proxy.Stop().always(() => {
-            start("sudo");
+            if (proxy.Methods) {
+                const ids = Object.keys(proxy.Methods);
+                if (ids.length == 1)
+                    start(ids[0]);
+                else {
+                    setMethods(ids);
+                    setMethod(ids[0]);
+                }
```

Comment:
These 4 added lines are not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-17536-20220713-085711-6af71fce-fedora-36-devel//Coverage/lcov/github-pr.diff.gcov.html)